components/infobox/extensions/commons/infobox_extension_game_appearances.lua

```diff
@@ -0,0 +1,46 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Extension/GameApperances
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Game = require('Module:Game')
+
+local MAX_NUMBER_OF_PLAYERS_IN_PLACEMENT = 10
+
+local Appearances = {}
+
+---@class GameAppearancesArgs
+---@field player string?
+
+----Provide a list of games compete by a player
+---@param args GameAppearancesArgs
+---@return string[] | nil
+function Appearances.player(args)
+	if not args or not args.player then return end
+
+	local conditions = Array.map(Array.range(1, MAX_NUMBER_OF_PLAYERS_IN_PLACEMENT), function(index)
```

Comment:
```suggestion
local DEFAULT_MAX_NUMBER_OF_PLAYERS_IN_PLACEMENT = 10

local Appearances = {}

----Provide a list of games compete by a player
---@param args {player: string?, numberOfPlayersInPlacement}?
---@return string[]?
function Appearances.player(args)
	if not args or not args.player then return end
	local numberOfPlayersInPlacement = args.numberOfPlayersInPlacement or DEFAULT_MAX_NUMBER_OF_PLAYERS_IN_PLACEMENT

	local conditions = Array.map(Array.range(1, numberOfPlayersInPlacement), function(index)
```

- imo no need to make the args a class
- make the max index of player in placement dynamic so that if we use this on a wiki which needs a higher index we can pass that via args